### DISCUSSION TITLE
The range of GBMA values

### DISCUSSION THREAD

**Discussion by yixingx** [mtex-toolbox/mtex#2151] [Ask Anything]
Dear MTEX Forum Experts,
I noticed that the limit of grain boundary misorientation angle (GBMA) in MTEX is [0,100], and I would like to know how to determine the range of GBMA values, especially for Mg alloys. Does it have to be less than 100°?

**Comment by kilir** [comment]
Hi,

> Does it have to be less than 100°?
I'm not sure if I understand what you want to say.

The maximum misorientation angle depends on the crystal symmetry
In order to determine the maximum angle, you can do the following:
```Matlab
cs = crystalSymmetry('hexagonal')
fr = fundamentalRegion(cs)
fr.maxAngle / degree
```


Hope this helps.
Cheers,
Rüdiger
